presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -166,7 +167,17 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                         ImmutableList.of(new CountConstantOptimizer()),
                         ImmutableSet.of(new SimplifyCountOverConstant())),
                 new WindowFilterPushDown(metadata), // This must run after PredicatePushDown and LimitPushDown so that it squashes any successive filter nodes and limits
-                new MergeWindows(),
+                new IterativeOptimizer(
+                                stats,
+                                ImmutableList.of(
+                                        new UnaliasSymbolReferences(),
+                                        new PruneIdentityProjections(),
+                                        new MergeWindows()),
```

Comment:
you ported only `MergeWindows` so in this list there should be only `MergeWindows`. Why do you have also USR and PIP?